### PR TITLE
[WIP] add features from #2163

### PR DESCRIPTION
**Pull request by devsnek**
add features from issue #2163 

closes #2163 
- [x] add append option `{{content words="20" append="..."}}`
- [ ] [WIP] change the way append is passed to the helper #2163 `{{#content words="20"}}APPENDTHIS{{/content}}`

### CODE REVIEW

**Comment by GusCaplan** [comment]
@ErisDS I think that this helper should stay simple and not have all that stuff that #5060 suggests. That way, themers have an option depending on what they want to do.

File: core/server/helpers/content.js [hunk 1]
```diff
@@ -19,14 +19,19 @@ content = function (options) {
         truncateOptions[key] = parseInt(truncateOptions[key], 10);
     });
 
+    try {
```
Comment (acburdine) [inline core/server/helpers/content.js]: Like I mentioned in your other PR, this isn't needed. Fix it in the same way.

**Comment by acburdine** [comment]
@GusCaplan This needs tests as well.

**Comment by GusCaplan** [comment]
@acburdine @ErisDS I have written helpers before, and usually you can get the content wrapped in a helper with self, but ghost passes the values differently. What is the ghost way?

**Comment by kirrg001** [comment]
@GusCaplan Hey 👋  Thanks for your effort on this 🙂 . This PR is now 1 year old. Can i close this?

**Pull request closed without merging**